### PR TITLE
update forecasts

### PR DESCRIPTION
**Pull request by nikosbosse**

### CODE REVIEW

**Review by seabbs — CHANGES_REQUESTED** [review]
Probably a good idea to investigate this a bit more as rbind is maybe just hiding the issue rather than helping here.

**Comment by nikosbosse** [comment]
I found something on the internet that it may have something to do with some internal way that NA values are handled. 
The columns are of same length and the resulting file it produced was fine (at least the submission looked good). 
I'll check again, but suspect this is something of a dead end...

**Comment by nikosbosse** [comment]
tested again. bind_rows worked well on my local machine... This is confusing!
I reverted to using bind_rows and suggest we see how it goes on Monday. Hopefully the problem has solved itself.

File: reports/update-website.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+Rscript data-raw/update.R
```
Comment (seabbs) [inline reports/update-website.sh:3]: this will update the forecast date. Is this what you want to do here?

Comment (nikosbosse) [reply reports/update-website.sh:3]: ah - no. not intentionally. At least I don't intend to do anything with the forecast date anywhere. 
But I guess it doesn't do any harm? This script runs twice - on Sunday (when the new data comes in) and on Monday (after the submission). At that point on Monday all Rt scripts are already done.

Comment (seabbs) [reply reports/update-website.sh:3]: unless we need to rerun them...... why are we updating data here anyway?

Comment (nikosbosse) [reply reports/update-website.sh:3]: Hm I guess you're right and we can just delete that line from the script - data should be up to date anyway

Comment (nikosbosse) [reply reports/update-website.sh]: I've deleted it now :)

**Review by seabbs — APPROVED** [review]

**Pull request merged**